ui/packages/consul-ui/ember-cli-build.js

```diff
@@ -88,18 +114,24 @@ module.exports = function(defaults, $ = process.env) {
   }
 
   //
-  trees.app = mergeTrees([
-    new Funnel('app', { exclude: excludeFiles })
-  ].concat(
-    apps.filter(item => exists(`${item.path}/app`)).map(item => new Funnel(`${item.path}/app`, {exclude: excludeFiles}))
-  ), {
-    overwrite: true
-  });
-  trees.vendor = mergeTrees([
-    new Funnel('vendor'),
-  ].concat(
-    apps.map(item => new Funnel(`${item.path}/vendor`))
-  ));
+  (
+    function(apps) {
+      trees.app = mergeTrees([
+        new Funnel('app', { exclude: excludeFiles })
+      ].concat(
+        apps.filter(item => exists(`${item.path}/app`)).map(item => new Funnel(`${item.path}/app`, {exclude: excludeFiles}))
+      ), {
+        overwrite: true
+      });
+      trees.vendor = mergeTrees([
+        new Funnel('vendor'),
+      ].concat(
+        apps.map(item => new Funnel(`${item.path}/vendor`))
+      ));
+    }
+  // consul-ui will eventually be a separate app just like the others
+  // at which point we can remove this filter/extra scope
+  )(apps.filter(item => item.name !== 'consul-ui'));
```

Comment:
Lines 132-134 where the only lines added here. This piece of code is to copy over our feature-flagged applications that live outside of the main application, as we added `consul-ui` as an 'application' we don't need to copy the files over for that 'application' as its the 'host application'. Therefore we just wanted to filter `consul-ui` out of the application list just for this bit of code. This avoids renaming `apps` to something like `appsWithoutConsulUI = apps.filter()` by just making a new temporary scope with `apps` overwritten which I prefer.